app/src/main/java/net/bible/android/view/activity/page/MainBibleActivity.kt

```diff
@@ -529,6 +529,8 @@ class MainBibleActivity : CustomTitlebarActivityBase(), VerseActionModeMediator.
 
     private fun chooseTab() {
         val tabs = tabStrings
+        if(tabs.size < 2) return
```

Comment:
I guess your haveTabs property would also do the same thing. Doesn't matter, this works too.